### PR TITLE
ci(lint): lint and test helm chart

### PR DESCRIPTION
**Pull request by TartanLeGrand**
🌟 This PR introduces a new GitHub Actions workflow for linting and testing Helm charts upon every pull request. It ensures all charts meet our quality standards before they can be merged into the main branch.

### 🛠 **Details**:
- **Environment**: Utilizes `ubuntu-latest` for running the jobs.
- **Key Actions**:
  - 📦 **Checkout**: Fetches the code with full history.
  - ⚓ **Set up Helm**: Configures Helm (v3.14.4) using Azure's Helm setup action.
  - 🐍 **Set up Python**: Prepares Python environment, targeting the latest release of the 3.x series.
  - 📊 **Chart Testing**: Uses Helm's chart-testing-action to perform list changes, lint, and install tests based on the changes detected in Helm charts.

### 🎯 **Workflow Triggers**:
- 🔄 The workflow is triggered on every pull request, ensuring that changes are tested before they reach the main branch.

### ⚙️ **Additional Configurations**:
- 🌐 **Kind Cluster**: Creates a Kubernetes-in-Docker (kind) cluster if changes are detected, providing an isolated environment for testing chart installations.

### 👀 **Reviewers**:
🔍 Please review the setup and the steps included in the workflow. Let's ensure our Helm charts are up to the standards and ready for deployment without issues.

### 🔄 **Impact**:
💪 This action will enhance our CI/CD pipeline by automating the testing process, catching potential issues early in the development cycle.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new linting and testing workflow for pull requests.

- **Chores**
  - Updated workflow trigger to run on all pull requests.
  - Updated maintainers' names and URLs in the Helm chart.
  - Improved comment formatting and naming conventions in `values.yaml`.
  - Specified the chart path in `helmfile.yaml`.

- **Style**
  - Enhanced configuration file readability and consistency by adjusting comment formatting and section indentation.

### CODE REVIEW

**Comment by TartanLeGrand** [comment]
https://github.com/8gears/n8n-helm-chart/issues/97 @Vad1mo :smile:

**Comment by Vad1mo** [comment]
Cool, thank you very much

**Comment by Vad1mo** [comment]
I'll take a look at it

File: .github/workflows/lint-test.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        uses: helm/chart-testing-action@v2.6.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> "$GITHUB_OUTPUT"
+          fi
+
```
[the remark this answers is not in the document]
Comment (TartanLeGrand) [reply .github/workflows/lint-test.yaml:39]: Deprecated

**Review by Vad1mo — APPROVED** [review]

**Pull request merged**